gallery/how_to/work_with_microtvm/micro_aot.py

```diff
@@ -24,8 +24,10 @@
 `Alan MacDonald <https://github.com/alanmacd>`_
 
 This tutorial is showcasing microTVM host-driven AoT compilation with
-a TFLite model. This tutorial can be executed on a X86 CPU using C runtime (CRT)
-or on Zephyr plarform on a microcontroller that supports Zephyr platform.
+a TFLite model. AoTExecutor reduces the overhead of parsing graph at runtime 
+compared to GraphExecutor. Also, we can have better memory management using Ahead 
+of time compilation. This tutorial can be executed on a x86 CPU using C runtime (CRT)
+or on Zephyr platform on a microcontroller that supports Zephyr platform.
```

Comment:
@mehrdadh I would say rather: "or on Zephyr platform on a microcontroller that is supported by Zephyr" or, shorter: "... or on a microcontroller/board supported by Zephyr".